ui/src/DataService.js

```diff
@@ -17,6 +17,7 @@ if (process.env.NODE_ENV !== 'production') {
   baseUrl = 'http://localhost:8080/http://localhost:4000/api';
   wsUrl = 'ws://localhost:4000/api';
 }
+baseUrl = 'http://localhost:4000/api';
```

Comment:
This should not be there IMO. The code above should handle `baseUrl` for both production (`make serve`) and debug (`./scripts/development.sh`).

Reply:
I left it there from debugging.